test/unit_tests/modules/util/web-storage-proxy.js

```diff
@@ -0,0 +1,262 @@
+'use strict';
+
+var chai = require( 'chai' );
+var sinon = require( 'sinon' );
+var expect = chai.expect;
+var jsdom = require( 'mocha-jsdom' );
+
+describe( 'web-storage-proxy', function() {
+  var webStorageProxy, sandbox, setItem, getItem, removeItem, setStorage;
+
+  jsdom();
+
+  before( function() {
+    webStorageProxy =
+      require( '../../../../src/static/js/modules/util/web-storage-proxy.js' );
```

Comment:
Move require to top of file. There's no need to require a module before every test, right? If there's state that needs to be cleared, clear that out in the before. Though if that's necessary it may indicate the module should be a class or have an init function.